### ISSUE TITLE
Check for valid species scientific names

### ISSUE THREAD

**Issue by victor-wildlife** [ocean-data-factory-sweden/kso-utils#143]
We should add a roadblock to report if the species in the species.csv have non valid scientific names. Maybe checking the names against https://www.checklistbank.org/

**Comment by jannesgg** [comment]
Good idea, they seem to have a nice API for doing this https://api.checklistbank.org/
